src/main/scala/dspjunctions/JtagAxi.scala

```diff
@@ -62,65 +68,60 @@ class JtagAxiMaster()(implicit p: Parameters) extends NastiModule()(p) {
     val ctrlAxi = new NastiIO
     val dataAxi = new NastiIO
   })
-
-  val tap = Module(new JtagTwoAxiMasterClocked(io.jtag.TCK.asClock, false.B)) // io.jtag.TRST))
-  io.jtag <> tap.io.jtag
-  tap.io.control.fsmAsyncReset := false.B
-
-  io.ctrlAxi <> tap.io.ctrlAxi
-  io.dataAxi <> tap.io.dataAxi
-}
-
-class JtagTwoAxiMasterClocked(modClock: Clock, modReset: Bool)(implicit val p: Parameters)
-  extends Module(override_clock=Some(modClock), override_reset=Some(modReset))
-  with HasNastiParameters {
-  
   val irLength = 4
+  val idcode = (BigInt(14), JtagIdcode(1, 1, 0x42))
```

Comment:
The generator will yell at you if you double-assign instruction codes. As far as I know there is no standardized instruction for idcode, but the TAP will always select idcode for scan on reset.